doc/source/docs/user_guide/reproject_fiona.rst

```diff
@@ -71,17 +71,16 @@ This example requires rasterio 1.2+ and GDAL 3+.
     from shapely.geometry import shape
 
     # load example data
-    world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
-    # Reproject to Mercator (after dropping Antartica)
-    world = world[(world.name != "Antarctica") & (world.name != "Fr. S. Antarctic Lands")]
+    nybb = geopandas.read_file(geodatasets.get_path('nybb'))
```

Comment:
We should probably still use a global dataset here to ensure we have actual antimeridian issues